2200033_박지환/문제4.py

```diff
@@ -0,0 +1,14 @@
+# 소득 불균형
+
+T = int(input())
+
+for i in range(T):
+    test_case = int(input())
+    earning_list = list(map(int, input().split()))
+
+    cnt = 0
+    for j in range(test_case):
+        if earning_list[j] <= sum(earning_list)/test_case: # 소득이 소득평균 이하일 때
+            cnt += 1
```

Comment:
```suggestion
    avg = sum(earning_list)/test_case
    for j in range(test_case):
        if earning_list[j] <= avg: # 소득이 소득평균 이하일 때
            cnt += 1
```
평균을 `for` 문 밖에서 한 번만 계산하는 방법도 있습니다! 😄

Reply:
그렇군요 !  참고하겠습니다ㅎㅎ😋